spec/acceptance/foreman_rh_cloud_spec.rb

```diff
@@ -6,6 +6,14 @@
   it_behaves_like 'an idempotent resource' do
     let(:manifest) do
       <<-PUPPET
+      file { '/etc/foreman-proxy':
+        ensure => directory,
+      }
+
+      group { 'foreman-proxy':
+        ensure => present,
+      }
```

Comment:
Why do we need a foreman-proxy in puppet-foreman? That feels like a horrible violation of principles where Foreman should never read files from foreman-proxy. Is this because of certs?

Reply:
I already mentioned before that I feel advisor engine belongs closer to a Smart Proxy because that's where all our external integration lives. We really need to figure out external services in a consistent way, especially if we expect more services to be added. Perhaps our Smart Proxy model isn't scalable, but I've made a mental note we need to discuss this. For now a note above it in the acceptance test is a "good enough" to allow us to proceed?

Reply:
Yeah, we're abusing a bit of the SP architecture, even tho I still think it's not *actually* a SP thing (e.g. you can't have multiple of it, which breaks the SP paradigm).

I'll add a note to the test, yes.